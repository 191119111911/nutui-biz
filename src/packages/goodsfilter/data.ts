export default {
  // 默认选中项
  selectedValue: {
    filterAttrs: [0],
    goodsAttrs: [{ id: 1, values: ["12690", "19306"] }]
  },
  filterAttrs: [{
    id: 0,
    name: '仅看有货'
  }, {
    id: 1,
    name: '京东物流'
  }, {
    id: 2,
    name: '货到付款'
  }, {
    id: 3,
    name: '放心购'
  }],
  // 价格区间
  priceRanges: [{
    low: '53',
    high: '132',
    desc: '14%选择',
    extra: {}
  }, {
    low: '132',
    high: '314',
    desc: '45%选择',
    extra: {}
  }, {
    low: '3141',
    high: '13291329',
    desc: '27%选择',
    extra: {}
  }],
  // 商品属性筛选
  goodsAttrs: [{
    title: '品牌',
    id: 1,
    values: [{
      id: "12690",
      name: "蒙牛",
      extra: {},
    }, {
      id: "19306",
      name: "伊利",
      extra: {},
    }, {
      id: "15139",
      name: "三元（SAN YUAN）",
      extra: {},
    }, {
      id: "575367",
      name: "麦趣尔",
      extra: {},
    }, {
      id: "6024",
      name: "德亚（Weidendorf）",
      extra: {},
    }, {
      id: "3492",
      name: "安佳（Anchor）",
      extra: {},
    }, {
      id: "376553",
      name: "认养一头牛",
      extra: {},
    }, {
      id: "1",
      name: "小西牛",
      extra: {},
    }, {
      id: "2",
      name: "澳牧",
      extra: {},
    }, {
      id: "3",
      name: "光明",
      extra: {},
    }, {
      id: "4",
      name: "旺旺",
      extra: {},
    }, {
      id: "5",
      name: "西域春",
      extra: {},
    }, {
      id: "6",
      name: "新希望",
      extra: {},
    }, {
      id: "7",
      name: "Bio-E",
      extra: {},
    }, {
      id: "8",
      name: "植选",
      extra: {},
    }, {
      id: "9",
      name: "百菲酪",
      extra: {},
    }, {
      id: "10",
      name: "新农",
      extra: {},
    }, {
      id: "11",
      name: "卡士",
      extra: {},
    }, {
      id: "12",
      name: "君乐宝",
      extra: {},
    }, {
      id: "13",
      name: "甘蒂牧场",
      extra: {},
    }, {
      id: "14",
      name: "爱氏晨曦",
      extra: {},
    }, {
      id: "15",
      name: "简颜轻",
      extra: {},
    }, {
      id: "16",
      name: "歌帝梵",
      extra: {},
    }, {
      id: "17",
      name: "瑞士莲",
      extra: {},
    }, {
      id: "18",
      name: "BEAUTYBUFFET",
      extra: {},
    }, {
      id: "19",
      name: "纽麦福",
      extra: {},
    }, {
      id: "20",
      name: "TAUPO PURE",
      extra: {},
    }, {
      id: "21",
      name: "西域美农",
      extra: {},
    }, {
      id: "22",
      name: "风车牧场",
      extra: {},
    }]
  }, {
    title: '类别',
    id: 2,
    values: [{
      id: "76437",
      name: "鲜牛奶",
      extra: {},
    }, {
      id: "83307",
      name: "低温酸奶",
      extra: {},
    }, {
      id: "101080",
      name: "调味鲜牛奶",
      extra: {},
    }, {
      id: "86611",
      name: "低温牛奶",
      extra: {},
    }]
  }, {
    title: '口味',
    id: 3,
    values: [{
      id: "21484",
      name: "水果味",
      extra: {},
    }, {
      id: "154960",
      name: "白桃味",
      extra: {},
    }, {
      id: "32593",
      name: "巧克力味",
      extra: {},
    }, {
      id: "9689",
      name: "原味",
      extra: {},
    }, {
      id: "9787",
      name: "草莓味",
      extra: {},
    }, {
      id: "114540",
      name: "红枣味",
      extra: {},
    }, {
      id: "82221",
      name: "香蕉味",
      extra: {},
    }, {
      id: "82690",
      name: "水蜜桃味",
      extra: {},
    }]
  }, {
    title: '钙含量',
    id: 4,
    values: [{
      id: "83057",
      name: "高钙",
      extra: {},
    }, {
      id: "131114",
      name: "非高钙",
      extra: {},
    }]
  }, {
    title: '蛋白质含量',
    id: 5,
    values: [{
      id: "9756",
      name: "高蛋白",
      extra: {},
    }, {
      id: "131113",
      name: "非高蛋白",
      extra: {},
    }]
  }],

  beforeSelected: (done: any, selectedValue: any) => {
    if (selectedValue.values.length > 2) {
      console.log('最多选4个')
    } else {
      done()
    }
  },

  addressList: [{
    id: 1,
    detail: '北京市朝阳区八里庄街道'
  }, {
    id: 2,
    detail: '钓鱼岛钓鱼岛全区'
  }, {
    id: 3,
    detail: '北京市大兴区科创十一街18号院京东大厦'
  }]
}