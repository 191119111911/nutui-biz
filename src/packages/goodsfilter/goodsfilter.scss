@import '../../styles/mixins/text-ellipsis.scss';

.nut-goods-filter {
  height: 100%;
  padding: 0px 0px 60px;
  border-radius: 12px;
  background: #fff;
  font-size: 12px;
  overflow: auto;

  &__chunk {
    padding: 0 18px;
    margin-top: 24px;

    &--address {
      display: flex;
      align-items: center;
      justify-content: space-between;
    }

    &__label {
      display: flex;
      align-items: center;
      justify-content: space-between;
      margin-right: 10px;
      font-size: 14px;
      font-weight: bold;
    }

    &__group {
      flex: 1;
      display: flex;
      align-items: center;
      justify-content: flex-end;

      &__address {
        max-width: 150px;
        overflow: hidden;
        text-overflow: ellipsis;
        white-space: nowrap;
      }

      &__modify {
        margin-left: 8px;
        color: $primary-color;
      }


    }
  }

  &__chunk__type {
    margin-top: 18px;
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;

    div {
      margin-bottom: 8px;
      width: 84px;
      height: 30px;
      line-height: 30px;
      text-align: center;
      background: rgba(242, 243, 243, 1);
      border: 1px solid rgba(242, 243, 243, 1);
      border-radius: 16px;
      font-size: 12px;

      &.active {
        background: rgba(240, 37, 15, 0.05);
        border: 1px solid rgba(242, 39, 12, 1);
        color: rgba(242, 39, 12, 1);
      }
    }

  }

  &__chunk__price {
    &--range {
      display: flex;
      width: 100%;
      justify-content: space-between;
      align-items: center;
      margin-top: 18px;

      &__item {
        position: relative;
        flex: 1;
        height: 30px;
        line-height: 30px;
        background: rgba(246, 246, 246, 1);
        border-radius: 16px;
        text-align: center;

        &--placeholder {
          display: inline-block;
          width: 40px;
          text-align: center;
          color: #999;
        }

        .input-num {
          position: absolute;
          left: 0;
          width: 100%;
          padding: 0 20%;
          text-align: center;
          height: 30px;
          border: 0;
          background: transparent;
          font-size: 12px;
          font-weight: bold;
          color: #1a1a1a;
        }
      }

      &__cable {
        margin: 0 5px;
        width: 10px;
        height: 1px;
        border: 1px solid rgba(151, 151, 151, 1);
      }
    }

    &--recommend {
      margin-top: 8px;
      display: flex;
      justify-content: space-between;

      &__item {
        display: flex;
        flex-direction: column;
        align-items: center;
        justify-content: center;
        width: 84px;
        height: 40px;
        line-height: 14px;
        padding: 0 12px;
        text-align: center;
        background: rgba(242, 243, 243, 1);
        border: 1px solid rgba(242, 243, 243, 1);
        border-radius: 18px;
        font-size: 12px;

        div {
          &:first-child {
            @include oneline-ellipsis()
          }

          &:last-child {
            transform: scale(90%);
            color: $text-color;
          }
        }
      }

    }
  }

  &__chunk__groups {
    padding-top: 18px;
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
    align-items: center;

    &.PT0 {
      padding-top: 0;
    }

    &--item {
      padding: 0 8px;
      width: 84px;
      margin-bottom: 8px;
      height: 30px;
      line-height: 30px;
      background: rgba(246, 246, 246, 1);
      border: 1px solid rgba(246, 246, 246, 1);
      border-radius: 16px;
      text-align: center;
      white-space: nowrap;
      overflow: hidden;
      text-overflow: ellipsis;
      font-size: 12px;
      color: $title-color;

      &.active {
        background: rgba(240, 37, 15, 0.05);
        border: 1px solid rgba(242, 39, 12, 1);
        color: rgba(242, 39, 12, 1);
      }

    }

    &--itemmore {
      width: 84px;
      height: 30px;
      line-height: 30px;
      margin-bottom: 8px;
      text-align: center;
      font-size: 12px;
      background: rgba(246, 246, 246, 1);
      border: 1px solid rgba(246, 246, 246, 1);
      border-radius: 16px;
      color: $title-color;

      svg {
        margin-left: 5px;
        width: 9px;
        height: 9px;
      }
    }

    &:after {
      content: "";
      display: block;
      width: 84px;
    }
  }


  &__chunk__gap {
    margin-top: 24px;
    height: 10px;
    width: 100%;
    background-color: #f2f3f3;
  }

  &__chunk__list {
    margin: 24px 0;
    padding: 0 18px;
    &--item {
      margin: 12px 0;
      &__top {
        display: flex;
        flex-direction: row;
        align-items: center;
      }
      &__title {
        font-size: 14px;
        font-weight: bold;
        color: #1a1a1a;
      }
      &__subTitle {
        flex: 1;
        font-size: 12px;
        padding: 0 8px 0 20px;
        color: #fa2c19;
        overflow: hidden;
        text-align: right;
        text-overflow: ellipsis;
        white-space: nowrap;
      }
      .nut-icon {
        font-size: 8px;
      }
    }
  }

  &__operate {
    position: absolute;
    bottom: 0;
    width: 100%;
    height: 50px;
    display: flex;
    justify-content: space-around;
    align-items: center;
    border-top: 1px solid rgba(243, 243, 243, 1);
    border-radius: 0 0 12px 12px;
    background: #fff;

    &__btn {
      width: 130px;
      height: 38px;
      border-radius: 25px;
      font-size: 15px;
      color: rgba(51, 51, 51, 1);
      text-align: center;
      line-height: 38px;

      &--reset {
        border: 1px solid rgba(155, 155, 155, 1);
      }

      &--confirm {
        color: rgba(255, 255, 255, 1);
        background: rgba(242, 39, 12, 1);
      }
    }
  }


  .nut-icon-location2 {
    margin-right: 8px;
  }

  .nut-collapse-item {
    .collapse-item {
      display: flex;
      align-items: center;
      justify-content: space-between;
      padding: 0;
      color: $title-color;
      line-height: normal;
      font-size: 14px;

      .collapse-title {
        font-size: 14px;
        font-weight: bold;
      }

      .subTitle {
        flex: 1;
        padding: 0 8px 0 20px;
        position: initial;
        margin: 0;
        font-size: 12px;
        color: $primary-color;
        text-align: right;
        @include oneline-ellipsis()
      }

      .collapse-icon {
        position: initial;
        margin: 0 !important;
      }

      &::after {
        display: none;
      }
    }

    .collapse-wrapper {
      .collapse-content {
        padding: 0;
      }
    }
  }

}