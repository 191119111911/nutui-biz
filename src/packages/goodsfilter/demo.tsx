import React, { useState } from 'react'
import { GoodsFilter } from './goodsfilter'
import { Cell } from '@nutui/nutui-react'
import data from './data'

const GoodsFilterDemo = () => {

  const [state, setState] = useState(data)
  const [visiable1, setVisiable1] = useState(false)
  const [visiable2, setVisiable2] = useState(false)
  const [visiable3, setVisiable3] = useState(false)
  const [visiable4, setVisiable4] = useState(false)
  return (
    <>
      <div className='demo'>
        <h2>基础用法</h2>
        <Cell onClick={() => { setVisiable1(true) }}>点击进行商品筛选</Cell>
        <GoodsFilter
          visiable={visiable1}
          priceRanges={state.priceRanges}
          goodsAttrs={state.goodsAttrs}
          filterAttrs={state.filterAttrs}
          onClose={() => { setVisiable1(false) }}
        ></GoodsFilter>
        <h2>自定义图标</h2>
        <Cell onClick={() => { setVisiable2(true) }}>点击进行商品筛选</Cell>
        <GoodsFilter
          visiable={visiable2}
          priceRanges={state.priceRanges}
          goodsAttrs={state.goodsAttrs}
          filterAttrs={state.filterAttrs}
          icon={'heart'}
          onClose={() => { setVisiable2(false) }}
        ></GoodsFilter>
        <h2>设置默认展示行数</h2>
        <Cell onClick={() => { setVisiable3(true) }}>点击进行商品筛选</Cell>
        <GoodsFilter
          visiable={visiable3}
          priceRanges={state.priceRanges}
          goodsAttrs={state.goodsAttrs}
          filterAttrs={state.filterAttrs}
          maxLine={3}
          onClose={() => { setVisiable3(false) }}
        ></GoodsFilter>
        <h2>点击事件</h2>
        <Cell onClick={() => { setVisiable4(true) }}>点击进行商品筛选</Cell>
        <GoodsFilter
          visiable={visiable4}
          priceRanges={state.priceRanges}
          goodsAttrs={state.goodsAttrs}
          filterAttrs={state.filterAttrs}
          onClose={() => { setVisiable4(false) }}
          onReset={() => { console.log('onReset')}}
          onConfirm={(res) => { console.log('onConfirm', res) }}
          onClickAddress={() => {
            console.log('onClickAddress')
          }}
          onSelectedAttrs={(attr: any, selected: boolean, selectedAttrs: any) => {
            console.log('onSelectedAttrs', attr)
            console.log('selected', selected)
            console.log('selectedAttrs', selectedAttrs)
          }}
          onSelectedPrice={(range: any) => {
            console.log('onSelectedPrice', range)
          }}
        ></GoodsFilter>
      </div>
    </>
  );
};

export default GoodsFilterDemo;
