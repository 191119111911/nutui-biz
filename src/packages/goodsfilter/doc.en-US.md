#  GoodsFilter

### Intro

It is suitable for filtering products on the search result page.

### Install

``` javascript
import { GoodsFilter } from '@nutui/nutui-biz';
```

## Code Demo

### Basic Usage
```js
import React, { useState } from 'react';
import { Cell } from '@nutui/nutui-react';
import { GoodsFilter } from '@nutui/nutui-biz';
import data from "./GoodsFilter.js";

const App = () => {

  const [state, setState] = useState(data)
  const [visiable1, setVisiable1] = useState(false)
  return (
    <>
      <div className='demo'>
        <h2>Basic Usage</h2>
        <Cell onClick={() => { setVisiable1(true) }}>Click to select goods</Cell>
        <GoodsFilter
          visiable={visiable1}
          priceRanges={state.priceRanges}
          goodsAttrs={state.goodsAttrs}
          filterAttrs={state.filterAttrs}
          maxLine={3}
          icon={'heart'}
          onClose={() => { setVisiable1(false) }}
          onReset={() => { console.log('onReset')}}
          onConfirm={(res) => { console.log('onConfirm', res) }}
          onClickAddress={() => {
            console.log('onClickAddress')
          }}
          onSelectedAttrs={(attr: any, selected: boolean, selectedAttrs: any) => {
            console.log('onSelectedAttrs', attr)
            console.log('selected', selected)
            console.log('selectedAttrs', selectedAttrs)
          }}
          onSelectedPrice={(range: any) => {
            console.log('onSelectedPrice', range)
          }}
        ></GoodsFilter>
      </div>
    </>
  );
};
```


## API

### Props

| Attribute         |     Description                  | Type   | Default           |
|--------------|----------------------------------|--------|------------------|
| visible        | Expand or not                         | boolean | `false`               |
| confirmText    | Display confirm text       | string |        `确定`      |
| resetText    | Display reset text       | string |      `重置`        |
| priceRangeTitle    | Display price ranges text      | string |     `价格区间`         |
| addressTitle    | Display address text      | string |        `配送地址`      |
| selectedAddress    | Display selected address text        | string |           -   |
| resetDisable | Disable the reset button | boolean | `false` |
| priceRanges | price ranges list     | Array |-  |
| filterAttrs    | filter attrs   | Array |       -       |
| goodsAttrs    | goods attrs       | Array |          -    |
| specStyle  | item style | CSSProperties | - |
| selectedSpecShow | Show selected attrs or not | boolean | `true` |
| maxLine    | Display lines in default | number | 2 |
| icon | icon | string | 'arrow-up' |
| bottom | Operations in the bottom | ReactNode | - |

### Events

| Name | Description           | Attrs     |
|--------|----------------|--------------|
| onClose | close the popup | - |
| onReset  | click the reset button | - |
| onConfirm  | click the confirm button | `res` |
| onClickAddress | click to change address | - |
| onSelectedAttrs  | click filters | `attr: any, selected: boolean, selectedAttrs: Array<any>` |
| onSelectedPrice  | click price ranges | `range: any` |
| onBeforeSelected   | before click goods attrs | `done: Function, selectedValue: any` |
| onSelectedGoodsAttr | click goods attrs | `attrs: any, value: any` |
