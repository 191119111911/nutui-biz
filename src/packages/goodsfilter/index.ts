import { GoodsFilter } from './goodsfilter'

export default GoodsFilter
