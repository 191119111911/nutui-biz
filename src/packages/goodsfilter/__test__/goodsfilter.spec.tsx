import React from 'react'
import { render, waitFor, fireEvent } from '@testing-library/react'
import '@testing-library/jest-dom'
import { GoodsFilter } from '../goodsfilter'
import data from '../data'

const state = data
test('Show GoodsFilter', async () => {
  const { asFragment } = render(
    <GoodsFilter
      visible={true}
      priceRanges={state.priceRanges}
      goodsAttrs={state.goodsAttrs}
      filterAttrs={state.filterAttrs}
      maxLine={3}
      icon={'heart'}
      onReset={() => { console.log('onReset') }}
      onConfirm={(res) => { console.log('onConfirm', res) }}
      onClickAddress={() => {
        console.log('onClickAddress')
      }}
      onSelectedAttrs={(attr: any, selected: boolean, selectedAttrs: any) => {
        console.log('onSelectedAttrs', attr)
        console.log('selected', selected)
        console.log('selectedAttrs', selectedAttrs)
      }}
      onSelectedPrice={(range: any) => {
        console.log('onSelectedPrice', range)
      }}
    ></GoodsFilter>
  )
  expect(asFragment()).toMatchSnapshot();
})

test('Custome icon', async () => {
  const { container } = render(
    <GoodsFilter
      visible={true}
      priceRanges={state.priceRanges}
      goodsAttrs={state.goodsAttrs}
      filterAttrs={state.filterAttrs}
      icon={'heart'}
    ></GoodsFilter>
  )
  const icon = container.querySelector('i.nut-icon-heart')
  expect(icon).toBeTruthy;
})

test('Click test', async () => {
  const onReset = jest.fn()
  const onClickAddress = jest.fn()
  const onConfirm = jest.fn()
  const onSelectedAttrs = jest.fn()
  const onSelectedPrice = jest.fn()
  const { container } = render(
    <GoodsFilter
      visible={true}
      priceRanges={state.priceRanges}
      goodsAttrs={state.goodsAttrs}
      filterAttrs={state.filterAttrs}
      onReset={onReset}
      onConfirm={onConfirm}
      onClickAddress={onClickAddress}
      onSelectedAttrs={onSelectedAttrs}
      onSelectedPrice={onSelectedPrice}
    ></GoodsFilter>
  )
  const address = container.querySelector('.nut-goods-filter__chunk__group__address')
  fireEvent.click(address as Element)
  expect(onClickAddress).toBeCalled

  const resetBtn = container.querySelector('.nut-goods-filter__operate__btn--reset')
  fireEvent.click(resetBtn as Element)
  expect(onReset).toBeCalled

  // click the first filter
  const filter = container.querySelector('.nut-goods-filter__chunk__type div')
  fireEvent.click(filter as Element)
  expect(onSelectedAttrs).toBeCalledWith({
    id: 0,
    name: "仅看有货"
  }, true, [{
    id: 0,
    name: "仅看有货"
  }])
  const activeFilter = container.querySelector('.nut-goods-filter__chunk__type .active')
  expect(activeFilter).toEqual(filter)

  // expand
  const length = container.querySelectorAll('.nut-goods-filter__chunk__groups--item').length
  expect(length).toEqual(20)
  const expand = container.querySelector('.nut-icon-arrow-up')
  fireEvent.click(expand as Element)
  const expandedLength = container.querySelectorAll('.nut-goods-filter__chunk__groups--item').length
  expect(expandedLength).toEqual(43)

  const goodsAttr = container.querySelector('.nut-goods-filter__chunk__groups--item')
  fireEvent.click(goodsAttr as Element)

  const priceRange = container.querySelector('.nut-goods-filter__chunk__price--recommend__item')
  fireEvent.click(priceRange as Element)

  const confirmBtn = container.querySelector('.nut-goods-filter__operate__btn--confirm')
  fireEvent.click(confirmBtn as Element)
  expect(onConfirm).toBeCalledWith({
    address: "",
    filterAttrs: [
      {
        id: 0,
        name: '仅看有货'
      }
    ],
    goodsAttrs: [
      {
        id: 1,
        values: ['12690'],
        isExpand: true
      }
    ],
    price: {
      high: "132",
      low: "53",
    },
  })

  // input number
  const input = container.querySelector('.input-num')
  fireEvent.input(input as Element, {
    target: {
      value: ""
    }
  })
  fireEvent.input(input as Element, {
    target: {
      value: "3"
    }
  })

  // cancel
  fireEvent.click(filter as Element)
  fireEvent.click(goodsAttr as Element)
  fireEvent.click(confirmBtn as Element)
  expect(onConfirm).toBeCalledWith({
    address: "",
    filterAttrs: [],
    goodsAttrs: [
      {
        id: 1,
        values: [],
        isExpand: true
      }
    ],
    price: {
      high: "132",
      low: "3",
    },
  })
})
