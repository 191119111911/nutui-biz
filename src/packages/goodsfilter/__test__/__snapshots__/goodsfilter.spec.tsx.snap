// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show GoodsFilter 1`] = `
<DocumentFragment>
  <div
    class="overlay-fade-enter-active nut-overlay"
    style="z-index: 1000; animation-duration: 0.3s;"
  />
  <div
    class="round nut-popup-right  nut-popup"
    style="width: 80%; height: 100%; z-index: 1000; animation-duration: 0.3s;"
  >
    <div
      class="nut-goods-filter"
    >
      <div
        class="nut-goods-filter__chunk nut-goods-filter__chunk--address"
      >
        <div
          class="nut-goods-filter__chunk__label"
        >
          配送地址
        </div>
        <div
          class="nut-goods-filter__chunk__group"
        >
          <i
            class="nutui-iconfont nut-icon nut-icon-location2 nut-goods-filter__chunk__group__icon"
            style="font-size: 12px; height: 12px; width: 12px;"
          />
          <div
            class="nut-goods-filter__chunk__group__address"
          >
            <span>
              您还没有选中的地址
            </span>
          </div>
          <div
            class="nut-goods-filter__chunk__group__modify"
          >
            修改
          </div>
        </div>
      </div>
      <div
        class="nut-goods-filter__chunk"
      >
        <div
          class="nut-goods-filter__chunk__type"
        >
          <div
            class=""
          >
            仅看有货
          </div>
          <div
            class=""
          >
            京东物流
          </div>
          <div
            class=""
          >
            货到付款
          </div>
          <div
            class=""
          >
            放心购
          </div>
        </div>
      </div>
      <div
        class="nut-goods-filter__chunk"
      >
        <div
          class="nut-goods-filter__chunk__label"
        >
          价格区间
        </div>
        <div
          class="nut-goods-filter__chunk__price"
        >
          <div
            class="nut-goods-filter__chunk__price--range"
          >
            <div
              class="nut-goods-filter__chunk__price--range__item nut-goods-filter__chunk__price--range__item--low"
            >
              <span
                class="nut-goods-filter__chunk__price--range__item nut-goods-filter__chunk__price--range__item--placeholder"
              >
                最低价
              </span>
              <input
                class="input-num"
                type="number"
                value=""
              />
            </div>
            <span
              class="nut-goods-filter__chunk__price--range__cable"
            />
            <div
              class="nut-goods-filter__chunk__price--range__item nut-goods-filter__chunk__price--range__item--high"
            >
              <span
                class="nut-goods-filter__chunk__price--range__item nut-goods-filter__chunk__price--range__item--placeholder"
              >
                最高价
              </span>
              <input
                class="input-num"
                type="number"
                value=""
              />
            </div>
          </div>
          <div
            class="nut-goods-filter__chunk__price--recommend"
          >
            <div
              class="nut-goods-filter__chunk__price--recommend__item"
            >
              <div>
                53-132
              </div>
              <div>
                14%选择
              </div>
            </div>
            <div
              class="nut-goods-filter__chunk__price--recommend__item"
            >
              <div>
                132-314
              </div>
              <div>
                45%选择
              </div>
            </div>
            <div
              class="nut-goods-filter__chunk__price--recommend__item"
            >
              <div>
                3141-13291329
              </div>
              <div>
                27%选择
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="nut-goods-filter__chunk__gap"
      />
      <div
        class="nut-goods-filter__chunk__list"
      >
        <div
          class="nut-goods-filter__chunk__list--item"
        >
          <div
            class="nut-goods-filter__chunk__list--item__top"
          >
            <div
              class="nut-goods-filter__chunk__list--item__title"
            >
              品牌
            </div>
            <div
              class="nut-goods-filter__chunk__list--item__subTitle"
            />
            <i
              class="nutui-iconfont nut-icon nut-icon-heart nut-goods-filter__chunk__list--item__icon"
            />
          </div>
          <div
            class="nut-goods-filter__chunk__groups"
          >
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              蒙牛
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              伊利
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              三元（SAN YUAN）
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              麦趣尔
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              德亚（Weidendorf）
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              安佳（Anchor）
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              认养一头牛
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              小西牛
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              澳牧
            </div>
          </div>
        </div>
        <div
          class="nut-goods-filter__chunk__list--item"
        >
          <div
            class="nut-goods-filter__chunk__list--item__top"
          >
            <div
              class="nut-goods-filter__chunk__list--item__title"
            >
              类别
            </div>
            <div
              class="nut-goods-filter__chunk__list--item__subTitle"
            />
            <i
              class="nutui-iconfont nut-icon nut-icon-heart nut-goods-filter__chunk__list--item__icon"
            />
          </div>
          <div
            class="nut-goods-filter__chunk__groups"
          >
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              鲜牛奶
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              低温酸奶
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              调味鲜牛奶
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              低温牛奶
            </div>
          </div>
        </div>
        <div
          class="nut-goods-filter__chunk__list--item"
        >
          <div
            class="nut-goods-filter__chunk__list--item__top"
          >
            <div
              class="nut-goods-filter__chunk__list--item__title"
            >
              口味
            </div>
            <div
              class="nut-goods-filter__chunk__list--item__subTitle"
            />
            <i
              class="nutui-iconfont nut-icon nut-icon-heart nut-goods-filter__chunk__list--item__icon"
            />
          </div>
          <div
            class="nut-goods-filter__chunk__groups"
          >
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              水果味
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              白桃味
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              巧克力味
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              原味
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              草莓味
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              红枣味
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              香蕉味
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              水蜜桃味
            </div>
          </div>
        </div>
        <div
          class="nut-goods-filter__chunk__list--item"
        >
          <div
            class="nut-goods-filter__chunk__list--item__top"
          >
            <div
              class="nut-goods-filter__chunk__list--item__title"
            >
              钙含量
            </div>
            <div
              class="nut-goods-filter__chunk__list--item__subTitle"
            />
            <i
              class="nutui-iconfont nut-icon nut-icon-heart nut-goods-filter__chunk__list--item__icon"
            />
          </div>
          <div
            class="nut-goods-filter__chunk__groups"
          >
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              高钙
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              非高钙
            </div>
          </div>
        </div>
        <div
          class="nut-goods-filter__chunk__list--item"
        >
          <div
            class="nut-goods-filter__chunk__list--item__top"
          >
            <div
              class="nut-goods-filter__chunk__list--item__title"
            >
              蛋白质含量
            </div>
            <div
              class="nut-goods-filter__chunk__list--item__subTitle"
            />
            <i
              class="nutui-iconfont nut-icon nut-icon-heart nut-goods-filter__chunk__list--item__icon"
            />
          </div>
          <div
            class="nut-goods-filter__chunk__groups"
          >
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              高蛋白
            </div>
            <div
              class="nut-goods-filter__chunk__groups--item"
            >
              非高蛋白
            </div>
          </div>
        </div>
      </div>
      <div
        class="nut-goods-filter__operate"
      >
        <div
          class="nut-goods-filter__operate__btn nut-goods-filter__operate__btn--reset"
        >
          重置
        </div>
        <div
          class="nut-goods-filter__operate__btn nut-goods-filter__operate__btn--confirm"
        >
          确定
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
